index.html

```diff
@@ -0,0 +1,180 @@
+                       <div class="schedule-mobile">
+                         <h3>2022.06.30(MON) ~ (SUT)</h3>
+                         <h5>@Bukavu South Kivu Province</h5>
+                       </div>
+                      </div>
+             </header>
+             <section>
+                 <article class="services">
+                   <div class="title">
+                    <h4>SERVICES PROVIDED</h4>
+                    <div class="bar"></div>
+                   </div>
+                    
+                    <div class="grid">
+                      <div class="cards">
+                          <img src="./images/consult.jpeg" alt="service1 image">
+                          <div>
+                               <h4>Consulting Meeting</h4>
+                               <p>
+                                   Lorem ipsum dolor sit amet consectetur, adipisicing elit.  
+                               </p>
+                          </div>
+                      </div>
+                      <div class="cards">
+                          <img src="./images/marketing1.jpg"  alt="service2 image">
+                          <div>
+                               <h4>Marketing</h4>
+                               <p>
+                                   Lorem ipsum dolor sit amet consectetur, adipisicing elit.  
+                                </p>
+                          </div>
+                      </div>
+                      <div class="cards">
+                           <img src="./images/gps.jpg" alt="service3 image">
+
+                          <div>
+                               <h4>GPS Tracker Solutions</h4>
+                               <p>
+                                  Lorem ipsum dolor sit amet consectetur, adipisicing elit.  
+                               </p>
+                           </div>
+                      </div> 
+                      <div class="cards">
+                          <img src="./images/acces.jpg" alt="service4 image">
+                           <div>
+                               <h4>Acces Control systems</h4>
+                               <p>
+                                  Lorem ipsum dolor sit amet consectetur, adipisicing elit.  
+                               </p>
+                           </div>
+                      </div> 
+                      <div class="cards">
+                        <img src="./images/cctv-system.jpg" alt="service4 image">
+                         <div>
+                             <h4>CCTV system</h4>
+                             <p>
+                                Lorem ipsum dolor sit amet consectetur, adipisicing elit.  
+                             </p>
+                         </div>
+                    </div> 
+                    </div>
+                    <div class="elements-short">
+                      <a href="#">SEE THE WHOLE SERVICES</a>
+                    </div>
```

Comment:
### **OPTIONAL**
Hi, @Jeanbulambo this review is marked optional because it was not mentioned by the previous reviewer, kindly implement the changes before you merge your code to help the recruiter understand that you actually fellow the project design.
Please kindly follow the required design on [Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
### **Behance Design**
![main](https://user-images.githubusercontent.com/55596970/176932683-6386cb50-c8da-412b-8302-4279d259f226.png)

### **Your design**
![yourFeat](https://user-images.githubusercontent.com/55596970/176932722-f7750acc-0179-49b5-8d57-f88b5252db16.png)